docs/content/docs/test/Run.md

```diff
@@ -1,39 +1,111 @@
 ---
-title: Введение
+title: Запуск тестов
 type: docs
 ---
 
 # Запуск тестов
 
-## IDE
+{{<avito page>}}
+
+В данном документе собраны способы запуска unit и instrumentation тестов. Описаны наборы тестов, которые запускаются
+автоматически, а также способы позволяющие запустить нужные тесты в ручном режиме.
+
+## Автоматически запускаемые тесты
+
+Существует несколько наборов тестов, запускаемых автоматически.
```

Comment:
It's obvious. You've made a great detailed structure. We can omit this.

Reply:
Removed